src/cow-react/modules/limitOrders/pure/Orders/OrdersTable.tsx

```diff
@@ -214,7 +216,7 @@ export function OrdersTable({
     document.body.dispatchEvent(new Event('mousedown', { bubbles: true }))
   }, [])
 
-  const isRowSelectable = selectedOrders !== null
```

Comment:
The problem was here. After some refactoring, `selectedOrders` cannot be null anymore